{{cookiecutter.project_name}}/src/{{cookiecutter.__project_slug}}/{% if cookiecutter.parameter_set=='True' %}my_parameters.py{% endif %}

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright (c) {{ cookiecutter.__year }} {{ cookiecutter.full_name }}. All rights reserved.
+
+{{ cookiecutter.project_name }}: {{ cookiecutter.project_short_description }}
+"""
+
+import pybamm
+import numpy as np
+
+# Extra dependencies may be imported here.
+
```

Comment:
could you add an example function and scalar parameter?

Reply:
I added a sample function with scalar parameters from the OKane2022 parameter set, let me know if that suffices